main.css

```diff
@@ -97,3 +99,125 @@ nav ul .email-icon {
     display: block;
   }
 }
+
+/* Styling for headline  */
+
+.headline {
+  display: flex;
+  width: 100vw;
+  height: 870px;
+  align-items: center;
+  background-color: #1c1a19;
+  background-image: url(./assets/images/Illustration-header.png);
+  background-position: top 0 right 0;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.headline .social {
+  order: 1;
+}
+
+.headline .social ul {
+  height: 180px;
+  margin-left: 36px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.headline .content {
+  order: 1;
+  width: 566px;
+  margin-left: 87px;
+}
+
+.headline .content h1 {
+  font-family: 'Crete Round', sans-serif;
+  color: #f55800;
+  font-size: 72px;
+}
+
+.headline .content h2 {
+  font-family: 'Inter', sans-serif;
+  font-size: 32px;
+  color: #fff;
+  font-weight: 500;
+}
+
+.headline .content p {
+  font-family: 'Inter', sans-serif;
+  font-size: 20px;
+  line-height: 32px;
+  color: #f4f5f7;
+  font-weight: 400;
+}
+
+.headline .background {
+  order: 1;
+  height: 100%;
+  min-width: 530px;
+}
+
+@media only screen and (max-width: 600px) {
+  .headline {
+    justify-content: center;
+    flex-direction: column;
+    background-image: url('./assets/images/illustration-header-mobile.png');
+    background-position: bottom 0 right 0;
+    background-repeat: no-repeat;
+    background-size: 190px;
+  }
+
+  .headline .background {
+    display: none;
+  }
+
+  .headline .content {
+    margin: 0 auto;
+    width: 328px;
+  }
+
+  .headline .content h1 {
+    font-size: 56px;
+    line-height: 64px;
+  }
+
+  .headline .content h2 {
+    margin-top: 12px;
+    font-family: 'Roboto', Arial, Helvetica, sans-serif;
+    font-weight: 700;
+    font-size: 24px;
+    line-height: 32px;
+  }
+
+  .headline .content p {
+    margin-top: 12px;
+    margin-bottom: 20px;
+    font-family: 'Poppins', Arial, Helvetica, sans-serif;
+    font-weight: 400;
+    font-size: 16px;
+    line-height: 24px;
+  }
+
+  .headline .social {
+    margin: 0 auto;
+    align-self: flex-start;
+    width: 420px;
+    height: 20px;
+    order: 2;
+  }
```

Comment:
Kindly here you can delete or change the `width: 420px;` role because it makes the page overflow.

Your design 
![image](https://user-images.githubusercontent.com/35535130/153338639-99c1fdbc-60ed-4e29-8cd1-8f76d050d1f2.png)